core/src/main/java/io/grpc/internal/InternalSubchannel.java

```diff
@@ -32,12 +32,13 @@
 import com.google.errorprone.annotations.ForOverride;
 import io.grpc.Attributes;
 import io.grpc.CallOptions;
+import io.grpc.ChannelLogger;
+import io.grpc.ChannelLogger.Level;
```

Comment:
I'd suggest not importing this and always referencing it via `ChannelLogger.Level`. Otherwise it is pretty confusing and you can't import `java.util.logging.Level`.